x-pack/plugins/fleet/public/applications/fleet/sections/agents/agent_requirements_page/components/install_command_utils.ts

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { PLATFORM_TYPE } from '../../../../hooks';
+
+export function getInstallCommandForPlatform(
+  platform: PLATFORM_TYPE,
+  esHost: string,
+  serviceToken: string,
+  policyId?: string,
+  fleetServerHost?: string,
+  isProductionDeployment?: boolean
+) {
+  let commandArguments = '';
+
+  if (fleetServerHost) {
+    commandArguments += `--url=${fleetServerHost} \\\n`;
```

Comment:
does this param need to be passed for Quick start mode?

Reply:
No it's not needed, I will fix that